Cargo.toml

```diff
@@ -23,7 +23,7 @@ lto = "thin"
 codegen-units = 32
 
 [workspace.dependencies]
-tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "e7b32ae0e9f44a2ed8b82683fb0c6527eff7002b", features = [
+tantivy = { git = "https://github.com/stuhood/tantivy.git", package = "tantivy", branch = "stuhood.top-n-acceptor", features = [
```

Comment:
I'll open a PR for this one if we like how this looks: https://github.com/paradedb/tantivy/compare/main...stuhood:tantivy:stuhood.top-n-acceptor